policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/ar_income_tax_before_non_refundable_credits_joint.yaml

```diff
@@ -4,17 +4,19 @@
   input:
     ar_taxable_income_joint: 5_100
     ar_low_income_tax_joint: 500
+    ar_low_income_tax_eligible: true
     state_code: AR
   output:
     #round(5_100 * 0.00)
-    ar_income_tax_before_non_refundable_credits_joint: 0
+    ar_income_tax_before_non_refundable_credits_joint: 500
```

Comment:
should this test return 0 in 2022?

Reply:
according to the `ar_taxable_income_joint`, the  `ar_low_income_tax_joint` should be zero, but since we assign the value of `ar_low_income_tax_joint` arbitrarily as 500, it returns 500 instead of 0. Do we need to set the low-income tax as 0?